Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -155,55 +153,90 @@ void ApplicationManagerBase::InitAssetProcessorManager()
     const AzFramework::CommandLine* commandLine = nullptr;
     AzFramework::ApplicationRequests::Bus::BroadcastResult(commandLine, &AzFramework::ApplicationRequests::GetCommandLine);
 
-    if(commandLine->HasSwitch("zeroAnalysisMode"))
+    struct APCommandLineSwitch
+    {
+        APCommandLineSwitch(const char* switchTitle, const char* helpText)
+            : m_switch(switchTitle)
+            , m_helpText(helpText)
+        {
+
+        }
+        const char* m_switch;
+        const char* m_helpText;
+    };
+
+    const APCommandLineSwitch kCommand_waitOnLaunch("waitOnLaunch", "Briefly pauses Asset Processor during initializiation. Useful if you want to attach a debugger.");
```

Comment:
Why the k prefix?

Reply:
I'll change to UpperCamelCase, I was misremembering coding standards, other projects I've been on used k as the prefix for constants.